Cargo.lock

```diff
@@ -4086,6 +4120,26 @@ dependencies = [
  "getrandom",
 ]
 
+[[package]]
+name = "rayon"
```

Comment:
It’s unfortunate to add a dependency like this for such a small feature. Is this not avoidable somehow?